checks/http/wix.rb

```diff
@@ -0,0 +1,28 @@
+module Intrigue
+module Ident
+module Check
+class Wix < Intrigue::Ident::Check::Base
+
+    def generate_checks(url)
+    [
+        {
+            :type => "fingerprint",
+            :category => "service",
+            :tags => ["COTS","Development"],
```

Comment:
Wix is hosted: https://monosnap.com/file/r3iqi35iD01eutNAbAeODSAE1Rvgpv

COTS should almost always be self-hosted by orgs

What makes this "Development" - is this a tool for developers?

Perhaps we should create a dictionary of tags to make selecting them more obvious. Realize it's not always simple.

Reply:
My fault I misinterpreted the COTS meaning in this case.

The development tag was because I tracked it down to their developer pages which made me think this was tied down to the apps on their App Market. I'm not thinking this is just their hosting request id. I have made the correction to fix that.

And I have changed it to a SaaS tag.